synapse/app/generic_worker.py

```diff
@@ -960,9 +961,7 @@ def start(config_options):
     # streams. Will no-op if no streams can be written to by this worker.
     hs.get_replication_streamer()
 
-    reactor.addSystemEventTrigger(
```

Comment:
Does it matter that this is going from `addSystemEventTrigger` -> `callWhenRunning`.

Reply:
I don't think so.

callWhenRunning looks like:
```python
    def callWhenRunning(self, _callable, *args, **kw):
        if self.running:
            _callable(*args, **kw)
        else:
            return self.addSystemEventTrigger('after', 'startup',
                                              _callable, *args, **kw)
```

so really this is just going from `addSystemEventTrigger("before", "startup", ...)` to `addSystemEventTrigger("after", "startup", ...)`. I don't *think* that matters.

Reply:
👍 I don't think that would matter.